styles/README.md

```diff
@@ -45,14 +45,6 @@ Note that all SASS modules in this project's package can be imported, by deeper
 
 Isolate your customizations within a single module in the consuming application SASS (avoid adding dependencies to this repository's SASS in multiple places for ease of maintenance later). See the `client` for how buttons are customized in SASS. It is the hope that customization will become a rarer exception as these patterns get more established.
 
-### Q: How much do I need to know about Atomic Design to get my job done here?
```

Comment:
I pulled this entire section because the only thing left is `atoms`, and that may not be long for this world.